src/pages/LinkList/LinkList.tsx

```diff
@@ -0,0 +1,9 @@
+import { LinksGrid, Header } from "./components";
+export const LinkList = () => {
+  return (
+    <>
+      <Header />
```

Comment:
The header could be in `<Main />` because it could be shared by other pages.

Reply:
main is not a component, it's an html tag

Reply:
Paso a español porque parece que no me expresé bien.

Estaba hablando de [`Main` component](https://github.com/FromDoppler/doppler-sms-webapp/blob/main/src/components/Main.tsx), aunque también podría ser en `App` u otro componente más general.

Lo que quería decir, es que el header no se va a usar solo en `LinkList` sino que se va a usar en casi cualquiera de las páginas de esta app, por eso lo dejaría afuera como hacemos en Doppler WebApp, o Editors WebApp.

Reply:
no entiendo muy bien, pero creo que no podemos predecir si va a ser un header compartido, según los mockups actuales no lo seria, pero creo que estas cosas se pueden resolver fácilmente cuando sea certero que podría reutilizarse.
Fuente del mockup actual:
https://www.figma.com/file/CbWMD9LxZAOL6hyWJpSdJL/Flujo-de-aprobaci%C3%B3n-SMS?node-id=5%3A4&t=y9zEl3A7YvOPb0H8-0

Reply:
Entiendo, no había notado que el header era específico de `pages/LinkList/components/Header/Header.tsx` y entendía que era _el header de la aplicación_, pero no es así. Me parece correcto que, al menos por ahora, se use desde la página de LinkList.

Igual, dale una mirada al patron de React Router que utiliza el [Outlet](https://reactrouter.com/en/main/components/outlet), puede ser una forma superadora de componer las páginas.

Reply:
Hablando con @GusBaamonde me explicaba que esta estructura que tenemos en la [style guide](https://cdn.fromdoppler.com/doppler-style-guide/documentation/main/breadcrumb-component.html) fue pensada teniendo en cuenta la accesibilidad y él esta al tanto de que el diseño del _**breadcrumb**_ va a cambiar.

Mi opinión es que usemos lo que no ofrece la Style Guide así los cambios se distribuyen en todo Doppler sin un esfuerzo mayor a futuro, ahí podemos inspeccionar el **_HTML_** en el navegador y podemos ver las clases y la estructura sugerida.